packages/glob/src/internal-glob-options.ts

```diff
@@ -0,0 +1,28 @@
+export interface IGlobOptions {
```

Comment:
I need to update the ADR, this diverges a little bit

The ADR specifies a class, I ended up making an interface instead since optional properties provides a better experience for consumers.